### ISSUE TITLE
Result of call to `done` is unused

### ISSUE THREAD

**Issue by Skornos** [mxcl/PromiseKit#813]
* PromiseKit version: **6.2.1**
* CocoaPods

Using even the example below generates compiler warning `Result of call to 'done(on:_:)' is unused`. Is this expected behaviour?

```
firstly {
    .value(1)
}.get { foo in
    print(foo, " is 1")
}.done { foo in
    print(foo, " is 1")    
}
```

**Comment by mxcl** [comment]
yes this is expected, we document it in four places that this is expected including our troubleshooting guide that is linked from the template for creating new issues and that you didn't read:

https://github.com/mxcl/PromiseKit/blob/master/Documentation/Troubleshooting.md#result-of-call-to-fooon_-is-unused
